sdks/java/io/hadoop-common/pom.xml

```diff
@@ -32,6 +32,17 @@
 
   <dependencies>
     <dependency>
+      <groupId>org.apache.beam</groupId>
+      <artifactId>beam-sdks-java-core</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
```

Comment:
Not sure why this is `provided`, how is this different from the one you removed from `io/hdfs` ?